components/resources/ChangeVersionDrawer.js

```diff
@@ -57,18 +61,46 @@ const ChangeVersionDrawer = (props) => {
     closeDrawer();
   };
 
+  useEffect(() => {
+    if (!isOpen) {
+      dispatch({
+        type: resourceActions.SET_VERSION_SEARCH_TERM,
+        data: SEARCH_ALL,
+      });
+    }
+  }, [isOpen]);
+
   return (
     <Drawer isOpen={isOpen} onClose={closeDrawer}>
+      <div className={styles.versionSelectionHeader}>
+        <p className={styles.versionSelectionName}>{resourceName}</p>
+        <p className={styles.versionSelectionInstructions}>
+          Select from the list below to see occurrences related to that version
+        </p>
+      </div>
+      <div className={styles.versionSearchContainer}>
+        <ResourceVersionSearchBar
```

Comment:
Not a part of these changes, but do we want to debounce this input? I like the fast feedback, but I could see triggering this on every input change potentially causing some issues.